Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md

```diff
@@ -51,5 +51,9 @@
     7. User needs to delete the PIM non SC-ALT role assignment(s) and migrate user's critical role assignment(s) to SC-ALT PIM after the script 
        execution.
 
+## FAQs:
+     Q.1. If i am getting this error "Unable to find type [Microsoft.Azure.Commands.Common.Authentication.AzureSession]" what i need to do ?
```

Comment:
Remove 'If'

Reply:
Or something like this,
While running the remediation script '<ScriptName>' I am getting error "Unable to find type [Microsoft.Azure.Commands.Common.Authentication.AzureSession]". How can I resolve this error?

Reply:
This file is specific to this script only so no need to mention the script name.